package/MDAnalysis/coordinates/DMS.py

```diff
@@ -70,7 +70,9 @@ def get_coordinates(self, cur):
         particles = cur.fetchall()
         return [(p['x'], p['y'], p['z']) for p in particles]
 
-    def get_particle_by_columns(self, cur, columns=['x', 'y', 'z']):
+    def get_particle_by_columns(self, cur, columns=None):
```

Comment:
@programmdesign, I totally understand that this is the conservative approach. Making it cleverer in the sense I mentioned is probably not even possible within reasonable effort. Just stating that this is one of the cases where we might want to not follow Cody's suggestion.